src/proto/mgmt/pool.proto

```diff
@@ -58,6 +58,18 @@ message PoolDestroyResp {
 	int32 status = 1; // DAOS error code
 }
 
+// PoolReintegrateReq supplies pool identifier and force flag.
+message PoolReintegrateReq {
+	string uuid = 1; // uuid of pool to add target up to
+	uint32 rank = 3; // target to move to the up state
+	repeated uint32 targets = 4; // target ranks
```

Comment:
Skipped 2, was that a mistake?

Reply:
yes, forgot to renumber them after removing something.